UpcomingMovies/UpcomingMovies/MainPage.xaml

```diff
@@ -0,0 +1,80 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:UpcomingMovies.ViewModels"
+             xmlns:behavior="clr-namespace:UpcomingMovies.Behaviors"
+             xmlns:converter="clr-namespace:UpcomingMovies.Converters"
+             xmlns:ffimageloading="clr-namespace:FFImageLoading.Forms;assembly=FFImageLoading.Forms"
+             x:Class="UpcomingMovies.MainPage"
+             Title="Upcoming Movies">
+
+    <ContentPage.Resources>
+        <ResourceDictionary>
+            <converter:ItemVisibilityEventArgsToItemConverter x:Key="ItemVisibilityConverter" />
+            <converter:SelectedItemChangedEventArgsToItemConverter x:Key="SelectedItemConverter" />
+            <converter:RankIntToRankColorConverter x:Key="RankToColorConverter" />
+        </ResourceDictionary>
+    </ContentPage.Resources>
+
+    <ContentPage.BindingContext>
+        
+    </ContentPage.BindingContext>
```

Comment:
No. Actually it was my fist implementation, but I move the binding to the code and fogot to remove these lines.